### ISSUE TITLE
chat: key expires soon, update app

### ISSUE THREAD

**Issue by jnelson** [keybase/client#9579]
A few members of a Keybase chat team who joined via invitation (no existing Keybase identity) have received alerts saying something like "key expiring soon, update app."

No one here knows what that means or what to do about it. The naive solution of trying to update the app (we're using the deb repo) doesn't work because no updates are available. (I suspect that's not what the message means, but I'm clueless.)

I did not find likely strings in GitHub's search. Does anyone know what this message means and what to do about it?

**Comment by maxtaco** [comment]
Cc: @oconnor663 

@jnelson can you run `keybase --version`?

**Comment by jnelson** [comment]
`keybase version 1.0.35-20171114170030+d28b65dbe` for at least one of them.

**Comment by jnelson** [comment]
@maxtaco Second user accidentally ran the other version version:
```
keybase -- version
Client:  1.0.35-20171031160055+fa845a3cc
Service: 1.0.35-20171031160055+fa845a3cc\
```

**Comment by maxtaco** [comment]
thanks for the feedback, there is a  minor bug here, @oconnor663  can explain.

if you're past `1.0.35-20171114`, you are good

**Comment by oconnor663** [comment]
This is a bug that should go away as soon as you `run_keybase` or restart. Sorry for the noise!

Details: Last night we pushed the config change to turn on the aggressive update message, to fix an issue with an expiring key. When I made that change, I misspelled the latest version, and the result was that _everyone_ was getting the warning, even after applying the update. We fixed the config within a few minutes, but unfortunately any client that made an API request during that window saved the "I am out of date" flag from the response. These clients will keep incorrectly printing an update warning forever, until the background service is restarted >.<
